src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -223,18 +223,18 @@ public void parseMedicalHistory_collectionWithValidTags_returnsMedicalHistorySet
 
     @Test
     public void parseMedicalHistory_emptyCollection_returnsEmptySet() throws Exception {
-        assertTrue(ParserUtil.parseMedicalHistories(Collections.emptyList()).isEmpty());
+        assertTrue(ParserUtil.parseMedicals(Collections.emptyList()).isEmpty());
     }
 
     @Test
     public void parseMedicalHistory_invalidValue_throwsParseException() {
-        assertThrows(ParseException.class, () -> ParserUtil.parseMedicalHistory(INVALID_MEDICAL_HISTORY));
+        assertThrows(ParseException.class, () -> ParserUtil.parseMedical(INVALID_MEDICAL_HISTORY));
     }
 
 
     @Test
     public void parseTag_emptyValue_throwsParseException() {
-        assertThrows(NullPointerException.class, () -> ParserUtil.parseMedicalHistories(null));
+        assertThrows(NullPointerException.class, () -> ParserUtil.parseMedicals(null));
```

Comment:
If this is a test case that we are keeping, the name of the test case should be `parseMedical` or `parseMedicalHistory`.